rust/gateway/README.md

```diff
@@ -0,0 +1,32 @@
+# gateway
+
+This crate houses the Firezone gateway.
+
+## Building
+
+You can build the gateway using: `cargo build --release --bin firezone-gateway`
+
+You should then find a binary in `target/release/firezone-gateway`.
+
+## Running
+
+To run the gateway:
+
+```
+firezone-gateway --portal_token <portal_token>
+```
+
+where `portal_token` is the token shown when creating a gateway group in the
+Firezone admin portal.
```

Comment:
Nit: Personally, I am fan of "one sentence per line" for all docs that are in Git as it aligns well with Git's model of diffing line-by-line.

Reply:
Hm, that might not be a bad idea, but this is auto-formatted on save by my editor. Does your editor automatically handle that?